config/coreboot-x230.config

```diff
@@ -1,4 +1,4 @@
-CONFIG_LOCALVERSION="heads"
+CONFIG_LOCALVERSION="0.2.1 heads"
```

Comment:
@Asiderr Is it required in order for the update to have greater version and apply? Can we at least heave it as a separate commit?